Velocity_Model/scripts/gen_basin_stats.sl

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
```

Comment:
I don't think this is the right repo to store slurm scripts.

Reply:
any suggestion?

Reply:
Let's have a chat over standup with thoughts from @jpa198 too.